zerver/openapi/zulip.yaml

```diff
@@ -4866,7 +4866,11 @@ paths:
         response. If all messages were marked as read, then the success response
         will return `"complete": true`.
 
-        **Changes**: Before Zulip 8.0 (feature level 211), if the server's
+        **Changes**: Deprecated; clients should use the [update personal message
+        flags for narrow](/api/update-message-flags-for-narrow) endpoint instead
+        as this endpoint may be removed in a future release.
+
+        Before Zulip 8.0 (feature level 211), if the server's
```

Comment:
I think we can add a `deprecated: true` on these in the OpenAPI spec too. We enforce that for parameters with the word "deprecated" apprearing in the description via our tooling, but apparently not at the top-level.

I think we should also say "will be removed" in this case, as we have concrete intent to remove, not just "may be removed".